controller/helper/process-survey-instances.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+/**
+ * @module controller/helper/process-survey-instances
+ */
+
+/**
+ * Takes in a Survey Instances and processes it to get Complience chart details
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} Complience chart data
+ */
+function processSurveyInstances (surveys) {
+    return {
+        labels: pickupSubmissionTime(surveys),
+        datasets: calculateTimeLeft(surveys)
+    };
+}
+
+/**
+ * Takes in a Survey Instances and get the dates to be shown on Complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} list of dates labels
+ */
+function pickupSubmissionTime (surveys) {
+    const dates = [];
+
+    for (let index = 0; index < surveys.length; index += 1) {
+      dates[index] = formatDate(surveys[index].startTime);
+    }
+    return dates;
+}
+
+/**
+ * Takes in a date as per DB format and process it to complience chart format
+ * @param {Object} date - DB formatted Date
+ * @returns {Object} formatted date
+ */
+function formatDate (date) {
+    let hours = date.getHours();
+    let minutes = date.getMinutes();
+    minutes = minutes < 10 ? '0'+minutes : minutes;
+    let strTime = hours + ':' + minutes;
+    return date.getMonth()+1 + "/" + date.getDate() + "/" + date.getFullYear() + "  " + strTime;
```

Comment:
Use `Moment.format` for this.
``` javascript
return moment(date).format('MM/DD/YYYY HH:mm');
```
would produce the format you want here.

Reference: http://momentjs.com/docs/#/displaying/format/